README.md

```diff
@@ -1,82 +1,7 @@
-# training-specific-template
+# CCDL Virtual Workshop: 2020 June Training
```

Comment:
```suggestion
---
title: CCDL Virtual Workshop: 2020 June Training
---
```

Reply:
This appears to break things for me locally - I'm pointed to the index when I navigate to the server address (but perhaps this is intended).

Reply:
I wonder if it is the colon...

Reply:
Yea that would make sense

Reply:
https://github.com/AlexsLemonade/2020-june-training/pull/5/commits/5c1822035c09f51ef4ca32d4acfb61d22092f707 did not appear to fix it